src/Stryker.Core/Stryker.Core.UnitTest/Initialisation/AssemblyReferenceResolverTests.cs

```diff
@@ -31,12 +31,13 @@ public void AssemblyReferenceResolver_ShouldAnalyzeProjectDependencies()
                     Output = @"  ExampleProject.Contracts -> C:\Repos\ExampleProject\ExampleProject\bin\Debug\netcoreapp2.0\ExampleProject.Contracts.dll
   ExampleProject -> C:\Repos\ExampleProject\ExampleProject\bin\Debug\netcoreapp2.0\ExampleProject.dll
   C:\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.netcore.app\2.0.0\ref\netcoreapp2.0\Microsoft.CSharp.dll;C:\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.aspnetcore.mvc.formatters.json\2.0.2\lib\netstandard2.0\Microsoft.AspNetCore.Mvc.Formatters.Json.dll"
+.Replace('\\', Path.DirectorySeparatorChar)
                 });
             metadataReferenceProviderMock.Setup(x => x.CreateFromFile(It.IsAny<string>()))
                 .Returns(() => null)
                 .Callback((string reference) => foundReferences.Add(reference));
 
-            string project = "C:/ProjectFolder/ExampleProject/ExampleProject.Test.csproj";
+            string project = @"C:\ProjectFolder\ExampleProject\ExampleProject.Test.csproj".Replace('\\', Path.DirectorySeparatorChar);
```

Comment:
I think it's clearer to use `Path.Combine`. This method is tailor made for this action.